backend/spec/controllers/api/v1/projects_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ProjectsController, :type => :controller do
+  describe "GET #index" do
+    it "returns all the projects, ordered by recency" do
+      older_project = FactoryGirl.create(:project,
+        created_at: Time.zone.now - 1.week)
+      old_project = FactoryGirl.create(:project,
+        created_at: Time.zone.now - 1.day)
+      oldest_project = FactoryGirl.create(:project,
```

Comment:
We can probably leave out the ordering stuff to make it less complicated. We can always add it during a later section. We'd probably want to order them alphabetically anyway.